dss/dynamodb/__init__.py

```diff
@@ -43,26 +48,26 @@ def put_item(*, table: str, hash_key: str, sort_key: Optional[str] = None, value
     db.put_item(**query)
 
 
-def get_item(*, table: str, hash_key: str, sort_key: Optional[str] = None, return_key: str = 'body') -> str:
+def get_item(*, table: str, hash_key: str, sort_key: Optional[str] = None) -> typing.Dict:
     """
     Get associated value for a given set of keys from a dynamoDB table.
-
-    Will determine the type of db this is being called on by the number of keys provided (omit
-    sort_key to GET a value from a db with only 1 primary key).
-
     :param table: Name of the table in AWS.
     :param str hash_key: 1st primary key that can be used to fetch associated sort_keys and values.
     :param str sort_key: 2nd primary key, used with hash_key to fetch a specific value.
-                         Note: If not specified, this will GET only 1 key (hash_key) and 1 value.
-    :param str return_key: Either "body" (to return all values) or "sort_key" (to return all 2nd primary keys).
-    :return: None or str
+    :return: item object from ddb (dict), with attribute types omitted
     """
+    return_value = {}
     query = {'TableName': table,
              'Key': _format_item(hash_key=hash_key, sort_key=sort_key, value=None)}
-    item = db.get_item(**query).get('Item')
+    try:
+        item = db.get_item(**query).get('Item')
+    except db.exceptions.ValidationException:
+        raise SchemaMismatch(f'Query schema {query} does not match table {table}')
```

Comment:
Is this exception clearer than `db.exceptions.ValidationException`?  I'd recommend keeping the original exception since it can be Googled, and maybe adding the custom message to it if it was originally unclear as to what was happening.